packages/dds/tree/src/core/edit-manager/editManager.ts

```diff
@@ -83,6 +81,7 @@ export class EditManager<
 
 	public constructor(
 		public readonly changeFamily: TChangeFamily,
+		public readonly localSessionId: SessionId,
```

Comment:
We should eventually unify this with the one in the compressor when it's in the runtime.

Reply:
Sure :)